src/components/ImportantNote/index.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
-import { withStyles, Typography } from '@material-ui/core';
+import { withStyles, IconButton, Tooltip, Typography } from '@material-ui/core';
 import cx from 'classnames';
 
 import styles from './styles';
 
-export const ImportantNote = withStyles(styles)(({ classes, heading, message, ...props }) => !!(heading && message) && (
-  <div {...props}>
-    <div className={classes.iconHeadingContainer}>
-      <i className={cx('icon iconfont icon-ic_info', classes.icon)} />
-      <Typography className={classes.heading}>{heading}</Typography>
+const ImportantNote = ({ classes, heading, message, ...props }) => (
+  heading && message && (
```

Comment:
It's the same logic. `!!` evaluates it if its truthy or not. `!!(heading && message)` = is saying check if heading and message are defined, if one of them are not then it will be false, otherwise true.